README.md

```diff
@@ -111,7 +116,7 @@ Example first deployment to a local cluster (from the project root directory):
 helm upgrade --install palisade . \
   --set global.persistence.dataStores.palisade-data-store.local.hostPath=${pwd}  \
   --set global.persistence.classpathJars.local.hostPath=${pwd}  \
-  --timeout=200s
+  --timeout=300s
```

Comment:
```suggestion
  --timeout=600s
```
as of recent changes by @developer01189998819991197253

Reply:
Done